### PR TITLE
Adding more tasks to MEG example

### PR DESCRIPTION
**Pull request by ja-che**

### CODE REVIEW

**Comment by bthirion** [comment]
LGTM. Simply make travis happy. Best,

**Comment by ja-che** [comment]
@agramfort is the example good to merge ?

File: examples/plot_meg_data_example.py [hunk 1]
```diff
@@ -4,10 +4,12 @@
```
Comment (agramfort) [inline examples/plot_meg_data_example.py:4]: can you add a paragraph here explaining what this example is about
and putting the ref to our paper?

File: examples/plot_meg_data_example.py [hunk 2]
```diff
@@ -93,34 +92,89 @@ def _compute_stc(zscore_active_set, active_set, evoked, forward):
...
+
+    # Handling average file
+    evoked = mne.read_evokeds(ave_fname, condition=condition,
+                              baseline=(None, 0))
+    evoked = evoked.pick_types('grad')
+
+    # Selecting relevant time window
+    evoked.plot()
+    t_min, t_max = 0.05, 0.1
+    t_step = 0.01
```
Comment (agramfort) [inline examples/plot_meg_data_example.py:133]: what is this tstep? I would add a decim parameter explaining it's for speed reason in the example.

File: examples/plot_meg_data_example.py [hunk 3]
```diff
@@ -93,34 +92,89 @@ def _compute_stc(zscore_active_set, active_set, evoked, forward):
...
+    noise_cov = mne.compute_covariance(epochs, rank='info', tmax=0.)
+
+    # Selecting relevant time window: focusing on early signal
+    evoked.plot()
+    t_min, t_max = 0.03, 0.05
+    t_step = 1.0 / 300
+
+    # We must reduce the whitener since data were preprocessed for removal
+    # of environmental noise leading to an effective number of 64 samples.
+    pca = True
```
Comment (agramfort) [inline examples/plot_meg_data_example.py:171]: did you try passing rank='info' in prepare_gain?

Comment (ja-che) [reply examples/plot_meg_data_example.py:171]: Yes and it is not reducing the whitener as desired: Solution are spurious for somato. 

'rank = None' works well.

**Comment by agramfort** [comment]
for the record https://69-259908992-gh.circle-artifacts.com/0/dev/auto_examples/plot_meg_data_example.html#sphx-glr-auto-examples-plot-meg-data-example-py

File: examples/plot_meg_data_example.py (hunk 2 above)
Comment (ja-che) [reply examples/plot_meg_data_example.py:133]: Not sure what would be the decim paramter ? Is it the same as:

`step = int(t_step * evoked.info['sfreq'])`

File: examples/plot_meg_data_example.py (hunk 1 above)
Comment (ja-che) [reply examples/plot_meg_data_example.py]: Done

**Comment by ja-che** [comment]
@agramfort, do you have other request regarding this PR ?

**Review by agramfort — COMMENTED** [review]
Can you share the link to the rendered doc so you simplify the feedback?

**Comment by ja-che** [comment]
@agramfort, I have improved this example. I believe it is nice now.
Please see [https://79-259908992-gh.circle-artifacts.com/0/dev/auto_examples/plot_meg_data_example.html](https://79-259908992-gh.circle-artifacts.com/0/dev/auto_examples/plot_meg_data_example.html).
WDYT ? Thank you !

**Review by agramfort — APPROVED** [review]
LGTM now !

thx @ja-che 

merge when green.

**Comment by agramfort** [comment]
🍺

**Comment by bthirion** [comment]
Congrats !

**Pull request merged**